### DISCUSSION TITLE
Getting error when adding  PnP search

### DISCUSSION THREAD

**Discussion by Abdelmadjid-Zenine** [microsoft-search/pnp-modern-search#3078] [Q&A]
I've installed PnP search v4 on sharepoint online search catalog, i gave api permissions.

I get the issue when i add the search results web part on my page 
below the error i'm getting this error in the console

![PnP error](https://github.com/microsoft-search/pnp-modern-search/assets/93162880/fc2d074e-4386-48ee-8a75-f9f90e3abadd)

Thanks for your help

**Comment by wobba** [comment]
Which version? How was it installed? New install or upgrade? Still an issue? Did you try to delete and re-add the sppkg?

Permissions are only needed for Graph functionality and should not matter here.
